android/src/org/ppsspp/ppsspp/NativeActivity.java

```diff
@@ -202,6 +206,13 @@ public void Initialize() {
         	// Get the optimal buffer sz
         	detectOptimalAudioSettings();
         }
+		powerManager = (PowerManager)getSystemService(Context.POWER_SERVICE);
+        if (Build.VERSION.SDK_INT >= 24) {
+			if (powerManager.isSustainedPerformanceModeSupported()) {
+				sustainedPerfSupported = true;
+				NativeApp.sendMessage("sustained_perf_supported","1");
```

Comment:
nit: `", "`.  We may have lost the war on consistent indenting, but...

-[Unknown]